AppLovinMAXDemo/Plugins/AppLovinMAX/AppLovinMAX.uplugin

```diff
@@ -1,9 +1,9 @@
 {
 	"FileVersion": 3,
-	"Version": 1000000,
-	"VersionName": "1.0.1",
+	"Version": 1010000,
+	"VersionName": "1.1.0",
```

Comment:
I usually break out changes and version bumps in different pushes (see RN: https://github.com/AppLovin/AppLovin-MAX-React-Native), just a bit cleaner, esp. if we end up stacking features before a push.